### DISCUSSION TITLE
Problem with some particular email addresses

### DISCUSSION THREAD

**Discussion by DomenicoFormoso** [PrestaShop/PrestaShop#37108] [Q&A] [answered]
On my Prestashop 8.0.5 site, customers who register with emails like a.b@c.xx after registering have problems both logging in and recovering their password. I noticed that the problem occurs for all users who have the dot on the left as their email address. How can I solve this?
thanks in advance

**Comment by kpodemski** [comment, marked as answer]
Hello @DomenicoFormoso 

It doesn't sound like an issue with PrestaShop but rather something that handles outgoing emails. I suggest contacting the server provider.
